src/Microsoft.NetFramework.Analyzers/UnitTests/TypesShouldNotExtendCertainBaseTypesTests.cs

```diff
@@ -347,97 +347,97 @@ End Class
         private static DiagnosticResult GetCSharpCollectionBaseResultAt(int line, int column, string declaredTypeName, string badBaseTypeName)
         {
             string message = string.Format(CultureInfo.CurrentCulture, MicrosoftNetFrameworkAnalyzersResources.TypesShouldNotExtendCertainBaseTypesMessageSystemCollectionsCollectionBase, declaredTypeName, badBaseTypeName);
-            return new DiagnosticResult(TypesShouldNotExtendCertainBaseTypesAnalyzer.Rule).WithLocation(line, column).WithArguments(message);
+            return VerifyCS.Diagnostic(TypesShouldNotExtendCertainBaseTypesAnalyzer.Rule).WithLocation(line, column).WithArguments(message);
```

Comment:
💡 Seems like this can be simplified based on lines below

```suggestion
            return VerifyCS.Diagnostic().WithLocation(line, column).WithArguments(message);
```